azure/v14.1.7/announcement.md

```diff
@@ -0,0 +1 @@
+**Workload cluster release v14.1.7 for Azure** is now available. This release upgrades Kubernetes to v1.19.12. 
```

Comment:
Ping @T-Kukawka - I'm not sure if there's anything more to say. AFAICS there's no major fixes between 1.19.10 - 1.19.12. Bug fixes for Azure, yes, but no CVEs that impact us nor any other critical changes. Is that enough for announcement?

Reply:
makes sense for me

Reply:
yeah sounds good 👍